src/components/molecules/PostCard.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Card,
+  CardContent,
+  Typography as MuiTypography,
+  styled,
+} from "@mui/material";
+import { Post } from "../../hooks/useFetchPosts";
+
+interface PostCardProps {
```

Comment:
There are no additional props beyond those in the `Post` interface. Therefore, there's no need to create another interface. Create a single interface file and reuse it in both files.

Reply:
ok got it, just a minute